CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -119,6 +126,12 @@ async def manual_report(state_file: UploadFile = File(None), cdc_file:  UploadFi
     res_file = os.path.join(app.dir, folder_name, id, "results.csv")
     process = await asyncio.create_subprocess_exec(sys.executable, os.path.join(app.dir, "compare.py"), '-c', cdc_save_to, '-s', state_save_to, '-o', res_file)
     await process.wait()
+    
+    if not os.path.exists(archive_path):
+        os.makedirs(archive_path)
+    
+    # Writing the newly created results file to the archive folder too
+    shutil.copy2(res_file, archive_path)
```

Comment:
Make sure we are either renaming the file before copying or storing each results.csv because with this implementation I believe that the results.csv will be overwritten every time a report is generated.